yarn_api_client/resource_manager.py

```diff
@@ -421,3 +422,72 @@ def cluster_change_application_priority(self, application_id, priority):
         path = '/ws/v1/cluster/apps/{appid}/priority'.format(appid=application_id)
 
         return self.request(path, 'PUT', data={"priority": priority})
+
+    def cluster_node_container_memory(self):
+        """
+        This endpoint allows clients to gather info on the maximum memory that
+        can be allocated per container in the cluster.
+        :returns: integer specifying the maximum memory that can be allocated in
+        a container in the cluster
+        """
+
+        maximum_container_memory = _get_maximum_container_memory(CONF_DIR)
+        return maximum_container_memory
+
+    def cluster_scheduler_queue(self, yarn_queue_name):
+        """
+        Given a queue name, this function tries to locate the given queue in the object
+        returned by scheduler endpoint.
+
+        The queue can be present inside a multilevel structure. This solution tries
+        to locate the queue using breadth-first-search algorithm.
+
+        :param yarn_queue_name:
+        :return: queue
```

Comment:
Should probably add something like _or `None` if not found_ and perhaps the expected datatype.